modules/calcite/src/test/java/org/apache/ignite/testsuites/PlannerTestSuite.java

```diff
@@ -69,6 +70,7 @@
     JoinWithUsingPlannerTest.class,
     ProjectFilterScanMergePlannerTest.class,
     IndexRebuildPlannerTest.class,
+    PlannerTimeoutTest.class
```

Comment:
Please add `,` to the end of line (it helps to merge branches without conflicts)